apps/site/app/[locale]/layout.tsx

```diff
@@ -15,8 +14,17 @@ import '@/styles/index.css';
 
 const fontClasses = classNames(IBM_PLEX_MONO.variable, OPEN_SANS.variable);
 
-const RootLayout: FC<PropsWithChildren> = async ({ children }) => {
-  const locale = await getLocale();
+type Props = {
+  params: Promise<{
```

Comment:
Can you inline this? In the sense of that this object can be all in one line? Promise<{ params: { locale: string } }>;